beacon-chain/blockchain/head.go

```diff
@@ -111,8 +112,9 @@ func (s *Service) saveHead(ctx context.Context, newHeadRoot [32]byte, headBlock
 			"oldSlot":            fmt.Sprintf("%d", headSlot),
 			"oldRoot":            fmt.Sprintf("%#x", oldHeadRoot),
 			"commonAncestorRoot": fmt.Sprintf("%#x", commonRoot),
+			"distance":           headSlot + newHeadSlot - 2*forkSlot,
```

Comment:
do we need to subtract 1?
Shouldn't the reorg distance for the below be 2?
```
99 - 100
    \- 101
```

Reply:
I would say that's 3, but I'm happy to subtract any constant, since it won't make the metric change in any meaning.

Reply:
I dont mind it, you can choose